src/client/firefox/commands.js

```diff
@@ -226,18 +236,17 @@ function removeBreakpoint(
   }
 }
 
-function setBreakpointCondition(
+function setBreakpointOptions(
   breakpointId: BreakpointId,
   location: SourceLocation,
-  condition: boolean,
-  noSliding: boolean
+  options: BreakpointOptions
 ) {
   const bpClient = bpClients[breakpointId];
   delete bpClients[breakpointId];
 
   const sourceThreadClient = sourceThreads[bpClient.source.actor];
   return bpClient
-    .setCondition(sourceThreadClient, condition, noSliding)
+    .setCondition(sourceThreadClient, transformOptionsToCondition(options))
```

Comment:
should we keep the `noSliding` argument?

Reply:
I removed this because BreakpointClient.prototype.setCondition doesn't actually take a third noSliding argument.  It also ignores the thread client argument, which is tidied up by the patch in bug 1520972.